packages/dds/tree/src/core/edit-manager/editManager.ts

```diff
@@ -113,119 +162,139 @@ export class EditManager<
 		// Note that option (A) would be a simple change to `addSequencedChange` whereas (B) would likely require
 		// rebasing trunk changes over the inverse of trunk changes.
 		assert(
-			this.localChanges.length === 0,
+			getPathFromBase(this.localBranch, this.trunk).length === 0,
```

Comment:
Can't we just check whether `this.localBranch === this.trunk`?

Reply:
Yes, good catch :)